lib/Froxlor/Dns/Dns.php

```diff
@@ -435,3 +430,4 @@ private static function generateDkimEntries($domain)
 		return $zone_dkim;
 	}
 }
+?>
```

Comment:
remove closing tags please...unnecessary and ugly :)

Reply:
Sorry - in my programming work everything which is opened must be closed - I will remove.